src/components/listbox/listbox.jsx

```diff
@@ -8,6 +8,7 @@ import styles from './listbox.css';
 export default function Listbox(props) {
   const [active, setActive] = useState(false);
   const [activeIndex, setActiveIndex] = useState(0);
+  const [selectedIndexes, setSelectedIndexes] = useState([]);
```

Comment:
```suggestion
// const [selectedIndexes, setSelectedIndexes] = useState([]);

  useImperativeHandle(ref, () => ({
    value: props.refs.filter(listOptionRef => listOptionRef.current.selected).map(selectedRef => selectedRef.current.value)
  }))
```

Reply:
This is a bit tricky. I'll do a write up with an analysis of whether this should be a good "web component" API or not.

Reply:
https://github.com/mavenlink/design-system/pull/140#discussion_r447188693

> To summarize the issue(s):

> This does not work when we are consuming DOM events (e.g. click), setting state, and then calling onChange because the ref.current is "stale."

> In order to get this to work, we need to consume DOM events, set state, and on state change call onChange -- which ensures the ref.current is latest. This is a good pattern to propagate.

However, we cannot use `value: props.refs...` because our listboxes will be paginated in the future. We cannot based listbox's value on the DOM because we might only have rendered part of the whole data.